src/main/scala/cromwell/binding/Call.scala

```diff
@@ -34,6 +35,27 @@ case class Call(alias: Option[String], task: Task, inputMappings: Map[String, Wd
    * @return Map[String, WdlType] representing each task-local input
    */
   def unsatisfiedInputs: Map[String, WdlType] = task.inputs.filterNot { case (k, v) => inputMappings.contains(k) }
-  
+
   override def toString: String = s"[Call name=$name, task=$task]"
+
+  /** Find all calls upon which this call immediately depends, i.e. the result of this
+    * does not include transitive dependencies.  Currently this only works for member
+    * access expressions with a literal LHS, e.g.:
+    * 
+    * {{{
+    *   call cgrep {
+    *     input: in_file=ps.procs
+    *   }
+    * }}}
+    * 
+    */
+  def prerequisiteCalls(): Traversable[Call] = {
```

Comment:
I'm more asking if you know they should always be a certain way. the one place it's being called is with .toSet, if logically the output of this should be a Set, it should return a Set. I'm asking because I don't know if that's the case though - is there a circumstance where you'd want duplicate Calls here?